bridge_adaptivity/config/settings/test.py

```diff
@@ -1,17 +1,15 @@
 # coding: utf-8
-
-from base import * # flake8: noqa: F401
+from base import *  # noqa: F401,F403
 
 TEST_RUNNER = 'config.test_runner.PytestTestRunner'
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'test.db',
-    }
-}
+try:
+    import secure
+    DATABASES = secure.DATABASES
+except ImportError:
+    import secure_example
```

Comment:
please use `as` this helps to avoid code duplication:
```
try:
     import secure
 except ImportError:
     import secure_example as secure
DATABASES = secure.DATABASES
```